homeassistant/components/hassio/sensor.py

```diff
@@ -0,0 +1,41 @@
+"""Sensor platform for Hass.io addons."""
+from typing import Callable, List
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import Entity
+
+from . import ADDONS_COORDINATOR
+from .const import ATTR_VERSION, ATTR_VERSION_LATEST
+from .entity import HassioAddonEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: Callable[[List[Entity], bool], None],
+) -> None:
+    """Sensor set up for Hass.io config entry."""
+    coordinator = hass.data[ADDONS_COORDINATOR]
+
+    entities = []
+
+    for addon in coordinator.data.values():
+        for attribute_name, sensor_name in (
+            (ATTR_VERSION, "Current Version"),
+            (ATTR_VERSION_LATEST, "Latest Version"),
```

Comment:
We use just "Version" for the installed version, and "Newest Version" for the available version in the UI, should we be consistent here?